<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemainingTilesView</class>
 <widget class="QWidget" name="RemainingTilesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="c" stretch="0,0,0,0">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tiles Left:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tilesLeftLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="discardedWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Discarded Tiles</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="discardedLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="remainingLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string notr="true">Base Game</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
